src/builtins/BuiltinTemporal.cpp

```diff
@@ -28,6 +28,40 @@
 
 namespace Escargot {
 #if defined(ESCARGOT_ENABLE_TEMPORAL)
+
+void checkTemporalCalendar(ExecutionState& state, const Value& thisValue, unsigned long argc)
```

Comment:
It looks like that this function is used in `BuiltinTemporal.cpp` file only.
So, please add `static` keyword to this function for better code generation.